### ISSUE TITLE
display projected victory points?

### ISSUE THREAD

**Issue by zeitlinger** [boardgamers/gaia-viewer#159]
would it make sense to show the projected victory points?

meaning
1. current points
2. - bid
3. + final points if all current resources were converted to c

**Comment by coyotte508** [comment]
Yes, in the last round, with also the final scoring tiles & round scoring tiles info

**Comment by zeitlinger** [comment]
> Yes, in the last round, with also the final scoring tiles & round scoring tiles info

I'm actually considering to show it before the last round as well - could there be any downside?

**Comment by coyotte508** [comment]
I guess it depends how it's done :p 

The concern I have is adding unnecessary info to the interface in the early rounds.

**Comment by zeitlinger** [comment]
Got it - some games don't show final vp on purpose,  to keep up suspense.

**Comment by coyotte508** [comment]
That's not what I meant. I'm not opposed to something unobtrusive showing a breakdown of all the projected VP changes - maybe down there with the advanced log or collapsible. I just don't want to clutter the interface.

It's a UX thing (which I'm not an expert of) and I guess it depends on the opinion of players, what amount of info they want to have at what stage of the game. Maybe it's helpful to have the projected total in 5th round as well, but I would find it distracting in the first two rounds. A good designer or UX designer could probably suggest something that would satisfy everybody.

**Comment by zeitlinger** [comment]
Good point - I'm thinking about an extra chart with vp graphs that could
also show stats about income per round - like starcraft.
This would be hidden behind a button,  so not clutter the interface.
